src/FSLibrary/MissionSorobanConfigUpgrades.fs

```diff
@@ -107,5 +107,7 @@ let sorobanConfigUpgrades (context: MissionContext) =
                       txMaxWriteBytes = Some(50000 * 2) }
                 (System.DateTime.UtcNow.AddSeconds(20.0))
 
-            formation.RunLoadgen coreSet context.GenerateSorobanUploadLoad
+            // There is a race condition where previously valid TXs in the TX queue may become invalid
```

Comment:
I don't think we need the new machinery: we can run the last loadgen run with skipLowFees=true, and then assert in the mission that 90%+ operations actually made it to the ledger (use `ledger.operation.apply` metric to extract the count of total ops applied, regardless of success of failure).